benchmark/realnest/01_aggregate-on-one-deep-struct-members.benchmark

```diff
@@ -0,0 +1,19 @@
+# name: benchmark/realnest/01_aggregate-on-one-deep-struct-members.benchmark
+# description: Aggregate functions on the struct, group by one parameter
+# group: [realnest]
+
+name aggregate-on-one-deep-struct-members
+group real_nest
+
+require json
+
+require httpfs
+
+cache real_nest.duckdb
```

Comment:
this `cache real_nest.duckdb` needs to be in every `.benchmark` file. Otherwise the data gets downloaded again for every benchmark